runtime/vm/CRIUHelpers.cpp

```diff
@@ -321,6 +327,8 @@ criuRestoreInitializeTrace(J9VMThread *currentThread, void *userData, const char
 				*nlsMsgFormat = j9nls_lookup_message(J9NLS_DO_NOT_PRINT_MESSAGE_TAG | J9NLS_DO_NOT_APPEND_NEWLINE,
 						J9NLS_VM_CRIU_RESTORE_INITIALIZE_TRACE_FAILED, NULL);
 				result = FALSE;
+			} else {
+				vm->checkpointState.flags |=  J9VM_CRIU_TRANSITION_TO_DEBUG_INTERPRETER;
```

Comment:
`criuRestoreInitializeTrace(currentThread)` finished w/o an error doesn't always indicate that a method is to be traced, i.e., it might not be required to `transitionToDebugInterpreter()`.
https://github.com/eclipse-openj9/openj9/blob/701c05ba23fcc83f7972ee7357a76e584c75661e/runtime/rastrace/trcengine.c#L1025-L1027
I think `J9VM_CRIU_TRANSITION_TO_DEBUG_INTERPRETER` can be enabled in this case, not the following 
https://github.com/eclipse-openj9/openj9/blob/701c05ba23fcc83f7972ee7357a76e584c75661e/runtime/rastrace/trcengine.c#L1032-L1033